doc/source/user_guide/ug_implementation.rst

```diff
@@ -122,6 +138,35 @@ and **global\_tx1.kmt** contain the :math:`\left<1^\circ\right>` POP
 tripole grid and land mask. These are binary unformatted, direct access,
 Big Endian files.
 
+The input grid file for the B-grid and CD-grid is identical.  That file
+contains each cells' HTN, HTE, ULON, ULAT, and kmt value.  From those
+variables, the longitude, latitude, grid lengths (dx and dy), areas,
+and masks can be derived for all grids.  Table :ref:`tab-gridvars` lists
+the primary prognostic grid variable names on the different grids.
+
+.. _tab-gridvars:
+
+.. table:: Primary CICE Prognostic Grid Variable Names
+
+   +----------------+-------+-------+-------+-------+
+   | variable       |   T   |   U   |   N   |   E   |
+   +================+=======+=======+=======+=======+
+   | longitude      |  TLON |  ULON |  NLON |  ELON |
+   +----------------+-------+-------+-------+-------+
+   | latitude       |  TLAT |  ULAT |  NLAT |  ELAT |
+   +----------------+-------+-------+-------+-------+
+   | dx             |  dxt  |  dxu  |  dxn  |  dxe  |
+   +----------------+-------+-------+-------+-------+
+   | dy             |  dyt  |  dyu  |  dyn  |  dye  |
+   +----------------+-------+-------+-------+-------+
+   | area           | tarea | uarea | narea | earea |
+   +----------------+-------+-------+-------+-------+
+   | mask (logical) | tmask | umask | nmask | emask |
+   +----------------+-------+-------+-------+-------+
+   | mask (real)    |  hm   | uvm   | npm   | epm   |
+   +----------------+-------+-------+-------+-------+
+
+
 In CESM, the sea ice model may exchange coupling fluxes using a
```

Comment:
Will we use this namelist variable 'gridcpl_file' for distinguishing between coupling on the C- and CD-grids?

Reply:
gridcpl_file is something totally different.  We have a new namelist, grid_system that can take on values of B, C, CD, etc as we move forward.

Reply:
I answered too quickly.  gridcpl_file allows a model to couple on a completely different grid.  In RASM we use this to coupled on an extended grid of the internal grid and we setup ice fraction to 0 at the extended grid points.  This helps the regional coupled model significantly.  The location of the forcing/coupling fields is totally different.  We'll need to create namelist to handle those options.  None of that is done yet, we still need to make some decisions about how we want to do that, and it's not critical for the initial implementation, but does need to be done at some point (it could even be after we have the CD implementation on the trunk).  For now, coupling/forcing is always assumed to be on the T grid.  Extending that to support other grids for subsets of forcing/coupling fields should not be difficult.